src/main/java/ogc/rs/database/DatabaseService.java

```diff
@@ -24,6 +24,8 @@ static DatabaseService createProxy(Vertx vertx, String address) {
 
     Future<List<JsonObject>>  getCollection(final String collectionId);
 
+    public Future<JsonObject> getJobStatus(String jobId, String userId);
```

Comment:
it's better to have a JSON object in the parameter instead of sending individual parameters.

Reply:
I have to disagree here - in case there are more parameters to pass, we can refactor to passing a class. But considering that there are only 2 values here, having them as well defined parameters instead of an opaque JSON object may be better for code readability.

Reply:
In the previous implementation of the jobs verticle, the entire requestBody (input JSON) was passed as a parameter. However, only the jobId and userId values from the requestBody were being utilized in the method. Therefore, I modified the method to accept these values as individual parameters (jobId and userId) instead of passing the entire requestBody.